### PR TITLE
Setting appropriate job status in hadoop2

### PR DESCRIPTION
**Pull request by avi-ram**

### CODE REVIEW

File: hraven-etl/src/main/java/com/twitter/hraven/etl/JobHistoryFileParserHadoop2.java [hunk 1]
```diff
@@ -421,16 +425,19 @@ private void processCounters(JSONObject eventDetails, String key) {
...
+            // look for job status
           // store it only if it's one of the terminal state events
-          if ((recType.equals(Hadoop2RecordType.JobFinished))
-              || (recType.equals(Hadoop2RecordType.JobUnsuccessfulCompletion))) {
-            this.jobStatus = eventDetails.getString(dataKey);
-          }
-        } else {
+          if (recType.equals(Hadoop2RecordType.JobFinished)) {
+            this.jobStatus = JOB_STATUS_SUCCESS;
+              System.out.println(this.jobStatus);
```
Comment (angadsingh) [inline hraven-etl/src/main/java/com/twitter/hraven/etl/JobHistoryFileParserHadoop2.java]: no sysouts

File: hraven-etl/src/test/java/com/twitter/hraven/etl/TestJobHistoryFileParserHadoop2.java [hunk 2]
```diff
@@ -90,7 +76,7 @@ public void testCreateJobHistoryFileParserCorrectCreation() throws IOException {
             JobHistoryKeys.JOB_STATUS.toString().toLowerCase()));
     
     assertNotNull(statusRecord);
-    assertEquals((String)statusRecord, JobHistoryFileParserHadoop2.JOB_STATUS_SUCCEEDED);
+    assertEquals((String)statusRecord, JobHistoryFileParserHadoop2.JOB_STATUS_SUCCESS);
```
Comment (angadsingh) [inline hraven-etl/src/test/java/com/twitter/hraven/etl/TestJobHistoryFileParserHadoop2.java:79]: how was the test passing earlier? did you change the history file in src/test/resources to match the new STATUS record name?

Comment (avi-ram) [inline hraven-etl/src/test/java/com/twitter/hraven/etl/TestJobHistoryFileParserHadoop2.java:79]: This is just a variable name change from JOB_STATUS_SUCCEEDED to JOB_STATUS_SUCCESS no logic change ...

**Comment by angadsingh** [comment]
@sandeepSamudrala kindly merge this

**Comment by sandeepSamudrala** [comment]
this pull req needs a rebase..
@avi-ram : please rebase and update the pull

**Pull request merged**